release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -696,6 +696,60 @@ digraph "org/package:0.1.0" {
 
 #### Improvements
 
+##### JSON to Record Converter
+Updated the conversion logic to handle certain scenarios more effectively.
```

Comment:
```suggestion
Updated the conversion logic to handle certain scenarios more effectively.
```
Rather than saying `certain scenarios`, as it sounds ambiguous, shall we specify them or just say `Improved the conversion logic....`?